{"name": "test_verify_failure_reason_saved", "status": "passed", "description": "\n        Verificar que el motivo de fallo se guarda correctamente\n        ", "start": 1760554132976, "stop": 1760554150519, "uuid": "839226f1-681b-4cbe-ba47-d2f083640e74", "historyId": "efe924dfd0fa933a1042828ca4f40b7f", "testCaseId": "efe924dfd0fa933a1042828ca4f40b7f", "fullName": "tests.test_rider_functionality.TestRiderFunctionality#test_verify_failure_reason_saved", "labels": [{"name": "tag", "value": "test_id('P2-013-VERIFY')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_rider_functionality"}, {"name": "subSuite", "value": "TestRiderFunctionality"}, {"name": "host", "value": "L43429"}, {"name": "thread", "value": "5772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_rider_functionality"}]}